src/sliding-sync-sdk.ts

```diff
@@ -779,7 +779,11 @@ export class SlidingSyncSdk {
         // This also needs to be done before running push rules on the events as they need
         // to be decorated with sender etc.
         room.addLiveEvents(timelineEventList, {
-            fromCache: fromCache,
+            // initial rooms are rooms that have been sent down to the client for the first time,
+            // which may include timeline events. We don't want these events to notify the user,
```

Comment:
Is this true? What if a room which isn't in your sliding window pings you?

Reply:
baaah stop asking awkward questions :D it won't work correctly. I guess what I really want here is `initial: true` rooms _and_ the JS SDK is unaware of the room. Will play.